unit_tests/testTimes/testTimes.f90

```diff
@@ -113,10 +105,23 @@ PROGRAM testTimes
 CALL testTimer%tic()
 CALL sleep(1)
 CALL testTimer%toc()
-ASSERT(ABS(testTimer%elapsedtime-1._SRK) < 0.05_SRK,'tic/toc')
-FINFO() 'testTimer%toc()= ',testTimer%elapsedtime
+totalElapsed=testTimer%elapsedtime
+ASSERT(totalElapsed > 0.0_SRK,'tic/toc')
+CALL testTimer%tic()
+CALL sleep(1)
+CALL testTimer%toc()
+ASSERT(testTimer%elapsedtime > totalElapsed,'tic/toc')
```

Comment:
Nitpicky, but given its a small file and we are trying to move this direction, `ASSERT_GT` and others. Afterwhich `FINFO` isn't required.

Reply:
Nah, good call.  I forgot those particular flavors existed

Reply:
Fixed